src/source/name.d

```diff
@@ -138,6 +137,13 @@ unittest {
 	}
 }
 
+unittest {
```

Comment:
Not a great place. What's above and bellow work together. Other wise, LGTM.

Reply:
oof yeah, I thought the above unittest was for the NameManager. Will move.

Reply:
fixed